### ISSUE TITLE
Update and align volunteers for next ITP on changes

### ISSUE THREAD

**Issue by kfklein15** [CodeYourFuture/Ops-Programme#249]
- We have made changes to the first module
- We need people to be aware what tasks are important
- We need to know who is doing what\
- Weekly ITP session (Amy runs on Tuesdays) inviting the 2 key people of each region - clearly state what is the most important and valuable now (i.e. code review, step approval)

**Comment by MarianaVazquezCYF** [comment]
can they not be updated on this when they do their edu training?

**Comment by AmyCYF** [comment]
Yes I agree @MarianaVazquezCYF - this should fall in Edu training
